apps/jsii-docgen/test/docgen/view/documentation.test.ts

```diff
@@ -1,8 +1,6 @@
 import * as path from "path";
 import { Documentation, Language } from "../../../src";
 
-jest.setTimeout(30 * 1000);
```

Comment:
Is the timeout not needed for this test?

Reply:
I believe it's now defined in the config file https://github.com/winglang/wing/pull/5781/files#diff-8384bc5ec390c8c8ac0c0c720db41b82ed8bb903afaccc803dd6cf660efa5315R7